Rakefile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
ya its updated in the skel I think, pretty sure I upgraded all the rubocop stuff recently.